Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -107,6 +107,50 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
             }
         }
 
+        [Theory]
+        [MemberData("ResponseModelTypes")]
```

Comment:
Oh, sorry. I just assumed this project used C# 6 by now (given that C# 7 has since been released), but I see the `LangVersion` elements from https://github.com/octokit/octokit.net/pull/820 are still there...